samples/IdentitySample.Mvc/Startup.cs

```diff
@@ -7,6 +7,10 @@
 using Microsoft.Framework.DependencyInjection;
 using Microsoft.Framework.Logging;
 using Microsoft.Framework.Logging.Console;
+#if DNX451
+using NLog.Config;
```

Comment:
Is the issue you mentioned with NLog support for scopes not being compatible with async still happening? Do you know if there is a bug and anyone following up on that?